spec/request/documents_spec.rb

```diff
@@ -45,13 +40,19 @@
     expect(args['tracked_item_id']).to be_nil
   end
 
-  context 'with locked PDF' do
-    let(:locked_file) do
-      fixture_file_upload(
-        "#{::Rails.root}/spec/fixtures/files/locked-pdf.pdf",
-        'application/pdf'
-      )
+  context 'with unaccepted file_type' do
+    let(:file) { fixture_file_upload('files/invalid_idme_cert.crt', 'application/pdf') }
```

Comment:
Is the 'application/pdf' type intentional here? It mismatches the crt file being passed in